### PR TITLE
normalize the code by replacing abs with fabs

### PR DESCRIPTION
**Pull request by inabao**

### CODE REVIEW

File: tests/test_index.cpp [hunk 1]
```diff
@@ -834,7 +834,7 @@ TEST_CASE("remove vectors from the index", "[ft][index]") {
             }
         }
         float recall_after = ((float)correct) / num_vectors;
-        REQUIRE(abs(recall_before - recall_after) < 0.001);
+        REQUIRE(std::fabs(recall_before - recall_after) < 0.001);
     } else {  // index that does not supports remove
```
Comment (LHT129) [inline tests/test_index.cpp:838]: why use fabs https://en.cppreference.com/w/cpp/numeric/math/fabs

**Review by wxyucs — APPROVED** [review]
lgtm

File: tests/test_index.cpp (hunk 1 above)
Comment (inabao) [reply tests/test_index.cpp:838]: done

**Review by LHT129 — APPROVED** [review]
LGTM

**Pull request merged**